workers/unity/Assets/Gdk/Core/Components/ComponentTranslation.cs

```diff
@@ -115,13 +93,37 @@ protected void RemoveComponents<T>(ref EntityCommandBuffer entityCommandBuffer,
                 }
             }
         }
+
+        protected static class TranslationErrors {
```

Comment:
This no longer exists in master. Please rebase before merging.

Reply:
This branch isn't targeting `master` 😉